lib/miq_pglogical/connection_handling.rb

```diff
@@ -15,8 +15,15 @@ def with_connection_error_handling
       end
 
       def pglogical(refresh = false)
```

Comment:
Technically this parameter is unused, and so rubocop will complain on it (i.e. change it to `_refresh`).  The bot filtered out that particular issue because this line is not part of the diff.

Reply:
I had `_refresh` originally when I was debating removing the now unused "refresh" completely which would include from the instance method `pglogical` below too.  I chose to be consistent with the other method so it's very clear they're linked and need to be cleaned up together.